### ISSUE TITLE
Accessibility: Creation UI live preview accessibility

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#42464]
Accessibility issues with the live preview:

- The `<aside>` has no label, meaning it only shows up as "complimentary" in the landmarks list
- When there is no live preview yet, the entire blurred "fake chart" in the background is not hidden from the screen reader and navigated before getting to the text "your live preview will show here"
- The button to _refresh_ the live preview is simply called "Live preview". The refresh functionality is only conveyed visually through the refresh icon.
